syncopy/tests/test_continuousdata.py

```diff
@@ -778,6 +785,157 @@ def test_sd_dataselection(self):
                             assert np.array_equal(cfg.out.data, selected.data)
                             time.sleep(0.05)
 
+    # test arithmetic operations
+    def test_sd_arithmetic(self):
+
+        # Create testing objects and corresponding arrays to perform arithmetics with
+        dummy = SpectralData(data=self.data,
+                             trialdefinition=self.trl,
+                             samplerate=self.samplerate,
+                             taper=["TestTaper_0{}".format(k) for k in range(1, self.nt + 1)])
+        dummyC = SpectralData(data=np.complex64(self.data),
+                              trialdefinition=self.trl,
+                              samplerate=self.samplerate,
+                              taper=["TestTaper_0{}".format(k) for k in range(1, self.nt + 1)])
+        ymmud = SpectralData(data=np.transpose(self.data, [3, 2, 1, 0]),
+                             trialdefinition=self.trl,
+                             samplerate=self.samplerate,
+                             taper=["TestTaper_0{}".format(k) for k in range(1, self.nt + 1)],
+                             dimord=SpectralData._defaultDimord[::-1])
+        dummy2 = SpectralData(data=self.data,
+                              trialdefinition=self.trl,
+                              samplerate=self.samplerate,
+                              taper=["TestTaper_0{}".format(k) for k in range(1, self.nt + 1)])
+        ymmud2 = SpectralData(data=np.transpose(self.data, [3, 2, 1, 0]),
```

Comment:
That this works surprises me.. isn't `self.data` some hdf5 container?

Reply:
Nope, in this case `self` refers to the class `TestSpectralData` which has an attribute `data`: https://github.com/esi-neuroscience/syncopy/blob/1896c31556e2da39aaf18766731368648e95b6f6/syncopy/tests/test_continuousdata.py#L631